docs/Troubleshooting.md

```diff
@@ -168,6 +170,23 @@ const epic = (action$: ActionsObservable<Actions>) =>
   );
 ```
 
+#### Redefine `ofType` with condition types (Typescript 2.8+)
+
+In your custom type definitions folder (see the [tsconfig typeRoots option](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types)), you can overload `ofType` by making a file and writing to it the following: 
+
+```typescript
+// redux-observable.d.ts (this comment is not needed).
+
+import { OperatorFunction } from "rxjs";
+import { Action } from "redux";
+
+declare module "redux-observable" {
+ function ofType<T extends Action<unknown>, K extends string>(
+  ...key: K[]
+ ): OperatorFunction<T, Extract<T, { type: K }>>;
+}
+```
```

Comment:
Oh it looks like perhaps your editor must have had 1 space indention. Can you go through and make sure they're all two spaces?